plugins/optimization-detective/class-od-url-metric.php

```diff
@@ -14,21 +14,27 @@
 /**
  * Representation of the measurements taken from a single client's visit to a specific URL.
  *
- * @phpstan-type RectData    array{ width: int, height: int }
- * @phpstan-type ElementData array{
- *                               isLCP: bool,
- *                               isLCPCandidate: bool,
- *                               xpath: string,
- *                               intersectionRatio: float,
- *                               intersectionRect: RectData,
- *                               boundingClientRect: RectData,
- *                           }
- * @phpstan-type Data        array{
- *                               url: string,
- *                               timestamp: int,
- *                               viewport: RectData,
- *                               elements: ElementData[]
- *                           }
+ * @phpstan-type RectData     array{ width: int, height: int }
+ * @phpstan-type ElementData  array{
+ *                                isLCP: bool,
+ *                                isLCPCandidate: bool,
+ *                                xpath: string,
+ *                                intersectionRatio: float,
+ *                                intersectionRect: RectData,
+ *                                boundingClientRect: RectData,
+ *                            }
+ * @phpstan-type Data         array{
+ *                                url: string,
+ *                                timestamp: int,
+ *                                viewport: RectData,
+ *                                elements: ElementData[]
+ *                            }
+ * @phpstan-type WebVitalData array{
+ *                                LCP: int,
+ *                                CLS: int,
+ *                                INP: int,
+ *                                TTFB: int,
+ *                            }
```

Comment:
I was wrong to use `int` for `timestamp`. The WebVitals metrics also should be floats in looking at some sample data I'm storing:

```json
{"webVitals":{"LCP":240.89999997615814,"CLS":0,"INP":null,"TTFB":83.19999998807907},"timestamp":1711666589.886095}
```

```suggestion
 * @phpstan-type Data         array{
 *                                url: string,
 *                                timestamp: float,
 *                                viewport: RectData,
 *                                elements: ElementData[]
 *                            }
 * @phpstan-type WebVitalData array{
 *                                LCP: float,
 *                                CLS: float,
 *                                INP: float|null,
 *                                TTFB: float,
 *                            }
```